### ISSUE TITLE
Run on the test data

### ISSUE THREAD

**Issue by kruthikakr** [mateuszbuda/brain-segmentation-pytorch#27]
Hi

If i need to run and test on single data , how to do with the inference.py ? instead of validation set . i need to run on single image .

**Comment by AK391** [comment]
@kruthikakr were you able to do this?

**Comment by AK391** [comment]
@mateuszbuda

**Comment by kruthikakr** [comment]
> @kruthikakr were you able to do this?

No , couldn't do this
